app/components/post_draft/send_handler/send_handler.tsx

```diff
@@ -129,6 +130,12 @@ export default function SendHandler({
     }, [intl, isTimezoneEnabled, channelTimezoneCount, doSubmitMessage]);
 
     const sendCommand = useCallback(async () => {
+        if (value.trim() === '/call end') {
+            await endCallAlert(serverUrl, intl, channelId);
+
+            // NOTE: fallthrough because the server may want to handle the command as well
```

Comment:
Can you please elaborate on this NOTE ? Why and how or is it just precaution?

Reply:
At the moment we're not doing anything with the `/call end` slash command on the server side of the plugin. But that could change in the future (maybe trigger cleanup of something, etc.). So we want to send the slash command just in case we need to handle it in the future.